src/components/core/MainNav/MainNav.tsx

```diff
@@ -0,0 +1,90 @@
+import { Location, NavLink, useLocation } from 'react-router-dom';
+import { HiOutlineHome } from 'react-icons/hi2';
+import { HiOutlineChatBubbleBottomCenterText } from 'react-icons/hi2';
+import { HiOutlineUser } from 'react-icons/hi2';
+import { HiOutlineUserGroup } from 'react-icons/hi2';
+import { HiOutlineUserPlus } from 'react-icons/hi2';
+import { HiOutlineCog6Tooth } from 'react-icons/hi2';
+import classes from './MainNav.module.scss';
+
+const MainNav = () => {
+  const location: Location = useLocation();
+
+  const isNavLinkActive = (to: string) => {
+    return location.pathname.includes(to);
+  };
+
+  return (
```

Comment:
I would suggest receiving elements to render here as list of objects with path and name, this way navigation is pretty much hard coded and adding new page is not great experience